<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestingWnd</class>
 <widget class="QDialog" name="TestingWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="pbnTest">
       <property name="text">
        <string>Тест</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTableView" name="tbvTest"/>
     </item>
     <item row="1" column="0">
      <widget class="QTableView" name="tbvSource"/>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="cbTestNum"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
